corpus/metadata/government.csv

```diff
@@ -1,65 +1,65 @@
-start,end,government
-1818-02-05,1840-05-16,Karl XIV Johans statsråd (till 1840)
-1840-03-28,1840-09-05,Regeringen Mauritz Posse I
-1840-09-05,1843-01-05,Regeringen Törnebladh
-1843-01-05,1844-12-28,Regeringen Gyllenhaal
-1844-12-28,1846-03-09,Regeringen Nordenfalk
-1846-03-23,1848-04-10,Regeringen Mauritz Posse II
-1848-04-10,1856-09-25,Regeringen Sparre
-1856-09-25,1858-04-07,Regeringen Günther
-1858-04-07,1870-06-03,Regeringen De Geer d.ä. I
-1870-06-03,1874-04-08,Regeringen Adlercreutz
-1874-05-04,1875-05-11,Regeringen Carleson
-1875-05-11,1876-03-20,Regeringen De Geer d.ä. II
-1876-03-20,1880-04-19,Regeringen De Geer d.ä. III
-1880-04-19,1883-06-13,Regeringen Posse
-1883-06-13,1884-05-16,Regeringen Thyselius
-1884-05-16,1888-02-06,Regeringen Themptander
-1888-02-06,1889-10-12,Regeringen Gillis Bildt
-1889-10-12,1891-07-10,Regeringen Åkerhielm
-1891-07-10,1900-09-12,Regeringen Boström I
-1900-09-12,1902-07-05,Regeringen von Otter
-1902-07-05,1905-04-13,Regeringen Boström II
-1905-04-14,1905-08-02,Regeringen Ramstedt
-1905-08-02,1905-11-07,Regeringen Lundeberg
-1905-11-07,1906-05-29,Regeringen Staaff I
-1906-05-29,1911-10-07,Regeringen Lindman I
-1911-10-07,1914-02-17,Regeringen Staaff II
-1914-02-17,1917-03-30,Regeringen Hammarskjöld
-1917-03-30,1917-10-19,Regeringen Swartz
-1917-10-19,1920-03-10,Regeringen Edén
-1920-03-10,1920-10-27,Regeringen Branting I
-1920-10-27,1921-02-23,Regeringen De Geer d.y.
-1921-02-23,1921-10-13,Regeringen von Sydow
-1921-10-13,1923-04-19,Regeringen Branting II
-1923-04-19,1924-10-18,Regeringen Trygger
-1924-10-14,1925-01-24,Regeringen Branting III
-1925-01-24,1926-06-07,Regeringen Sandler
-1926-06-07,1928-10-02,Regeringen Ekman I
-1928-10-02,1930-06-07,Regeringen Lindman II
-1930-06-07,1932-08-06,Regeringen Ekman II
-1932-08-06,1932-09-24,Regeringen Hamrin
-1932-09-24,1936-06-19,Regeringen Hansson I
-1936-06-19,1936-09-28,Regeringen Pehrsson-Bramstorp
-1936-09-28,1939-12-13,Regeringen Hansson II
-1939-12-13,1945-07-31,Regeringen Hansson III
-1945-07-31,1946-10-06,Regeringen Hansson IV
-1946-10-11,1951-10-01,Regeringen Erlander I
-1951-10-01,1957-10-31,Regeringen Erlander II
-1957-10-31,1969-10-14,Regeringen Erlander III
-1969-10-14,1976-10-08,Regeringen Palme I
-1976-10-08,1978-10-18,Regeringen Fälldin I
-1978-10-18,1979-10-12,Regeringen Ullsten
-1979-10-12,1981-05-22,Regeringen Fälldin II
-1981-05-22,1982-10-08,Regeringen Fälldin III
-1982-10-08,1986-03-12,Regeringen Palme II
-1986-03-12,1990-02-27,Regeringen Carlsson I
-1990-02-27,1991-10-04,Regeringen Carlsson II
-1991-10-04,1994-10-07,Regeringen Carl Bildt
-1994-10-07,1996-03-22,Regeringen Carlsson III
-1996-03-22,2006-10-06,Regeringen Persson
-2006-10-06,2014-10-03,Regeringen Reinfeldt
-2014-10-03,2019-01-19,Regeringen Löfven I
-2019-01-21,2021-07-09,Regeringen Löfven II
-2021-07-09,2021-11-30,Regeringen Löfven III
-2021-11-30,,Regeringen Andersson
\ No newline at end of file
+start,end,government,wikidata
```

Comment:
I'm thinking we should try to coordinate the wikidata_id column @ninpnin do you know what we use in other places? I think it is wikidata_id? That make more sense to me?